src/wmtk/autogen/tet_mesh/get_tuple_from_simplex_local_id.hpp

```diff
@@ -0,0 +1,87 @@
+#pragma once
+#include <cassert>
+#include <wmtk/Tuple.hpp>
+#include "autogenerated_tables.hpp"
+#if !defined(_NDEBUG)
+#include "is_ccw.hpp"
+#endif
+
+namespace wmtk::autogen::tet_mesh {
+/*
+namespace {
+template <size_t Dim>
+inline Tuple get_tuple_from_simplex_local_id_T(int8_t local_id, int64_t global_id = 0)
+{
+const auto& arr = autogen::tet_mesh::auto_3d_table_complete_edge[local_id];
+assert(arr[Dim] == local_id);
+const auto& [lvid, leid, lfid] = arr;
+
+
+if (lvid < 0 || leid < 0 || lfid < 0) {
+    throw std::runtime_error("tuple_from_id failed");
+}
+Tuple tuple = Tuple(lvid, leid, lfid, global_id);
+// accessor as parameter
+assert(is_ccw(tuple)); // is_ccw also checks for validity
+return tuple;
+}
+} // namespace
+*/
+
+inline Tuple get_tuple_from_simplex_local_vertex_id(int8_t local_id, int64_t global_id = 0)
+{
+    const auto& arr = autogen::tet_mesh::auto_3d_table_complete_vertex[local_id];
+    const auto& [lvid, leid, lfid] = arr;
+    assert(lvid == local_id);
+
+
+    if (lvid < 0 || leid < 0 || lfid < 0) {
```

Comment:
Is that even possible? That sounds like something that should be checked for in a test and not in here.